### PR TITLE
Harvester: typology field is inputable if get nodes failed

### PR DESCRIPTION
**Pull request by DaiYuzeng**
### Occurred changes and/or fixed issues

- https://github.com/harvester/harvester/issues/2567
- https://github.com/harvester/harvester/issues/2598

### CODE REVIEW

**Review by n313893254 — APPROVED** [review]
LGTM

File: shell/machine-config/harvester.vue [hunk 1]
```diff
@@ -169,6 +165,14 @@ export default {
...
         this.value.diskSize = '40';
       }
+
+      try {
+        const { data: nodes } = await this.$store.dispatch('cluster/request', { url: `${ url }/${ NODE }s` });
+
+        this.allNodeObjects = nodes;
+      } catch (err) {
+        this.allNodeObjects = [];
+      }
```
Comment (WuJun2016) [inline shell/machine-config/harvester.vue:175]: I think this part should also be put inside` if (clusterId && isImportCluster)`，  The external cluster does not need to request the node api.

**Review by WuJun2016 — CHANGES_REQUESTED** [review]

File: shell/machine-config/harvester.vue (hunk 1 above)
Comment (WuJun2016) [reply shell/machine-config/harvester.vue:175]: other LGTM.

**Review by WuJun2016 — CHANGES_REQUESTED** [review]
Question:
1. If is a importCluster, the `Pods in these namespaces:` should be selectable。
2. Which cluster should be the source of namespace ?

**Review by WuJun2016 — APPROVED** [review]
LGTM

**Comment by WuJun2016** [comment]
@n313893254  Can you help to review it again?  thanks.

**Pull request merged**